deployments/helm/nvidia-device-plugin/templates/daemonset-mps-control-daemon.yml

```diff
@@ -193,12 +193,11 @@ spec:
       volumes:
       - name: mps-root
         hostPath:
-        # TODO: This should be /var/run/nvidia/mps
-          path: /var/lib/kubelet/device-plugins/mps
+          path: {{ .Values.mps.root }}
```

Comment:
Question -- I notice that `mps-root` is mounted at `/driver-root/mps` in the container. Does this mean that the MPS root will be visible on the host at the configured `mpsRoot` (e.g. `/run/nvidia/mps`) and at `${driverRoot}/mps` (e.g. `/run/nvidia/driver/mps`)?

Reply:
Running in my kind cluster, it would appear as if there is indeed a folder created at `/driver-root/mps` on the host. (`/mps` in this case). There is *also* a folder created at `/run/nvidia/mps` on the host -- as expected -- and this is the folder that contains the `shm` directory and the per-resource `pipe` and `log` directories.

Reply:
Okay thanks.